### ISSUE TITLE
Metas not shown on notifications

### ISSUE THREAD

**Issue by TheOnlyNurse** [florent37/Flutter-AssetsAudioPlayer#163]
**Flutter Version**

My version : 1.17.2

**Lib Version**

My version : 2.0.0+5

**Platform (Android / iOS / web) + version**

Platform : Android

**Describe the bug**

Metas are not shown on playback notifications. However once updated [through Audio.file.updateMetas()] song information does appear. I am quite new to your package and flutter in general, so it could be simply a problem on my end.

**Small code to reproduce**
```
final audio = Audio.file(response, metas: Metas(title: 'hello world'));
```

Doesn't show information:
```dart
      await _assetsAudioPlayer.open(
        audio,
        showNotification: true,
      );
```

Does show:
```dart
audio.updateMetas(player: _assetsAudioPlayer, title: 'hello world',)
```

**Comment by florent37** [comment]
The first should work, the second is just for update, it's a bug on the lib I think

I'll try maybe it's only because you have a title, but it should work, I'll fix it, thanks for the issue

**Comment by florent37** [comment]
ok....

it's my fault, I let you find ^^

<img width="336" alt="Capture d’écran 2020-06-05 à 17 19 29" src="https://user-images.githubusercontent.com/5754972/83893841-c99db680-a750-11ea-99e4-69c224eeca55.png">

**Comment by florent37** [comment]
( metas on file is not used... 😄 )

**Comment by florent37** [comment]
fixed, please try the `^2.0.0+6`
